### ISSUE TITLE
Error when copying .app into applications.

### ISSUE THREAD

**Issue by orbifx** [wereturtle/ghostwriter#83]
Running:

    sudo cp ./build/release/ghostwriter.app /Applications

results in error:

    cp: ./build/release/ghostwriter.app is a directory (not copied).

Maybe add the `-R` flag? Not sure if that is normal for Mac installations. My work-around was to drag and drop the .app produced from the file manager into the applications.

**Comment by wereturtle** [comment]
I'm assuming you're trying to run `make install`, correct?  The Makefile is something that Qt is autogenerating from the ghostwriter.pro file.  the project (.pro) file doesn't explicitly have any cp command.  That is something that Qt did when it created the Makefile.

Out of curiosity, what Qt version are you using?

Also, can you try building and installing [FocusWriter](http://github.com/gottcode/focuswriter)?  ghostwriter lifted from its .pro file, and I'm curious if it gives you the same issue.

Unfortunately, I do not own a Mac, so I cannot do much to help you.  If you happen to find a solution (for the .pro file), please feel free to submit a pull request so that other Mac users can benefit.

**Comment by orbifx** [comment]
I didn't use make install, I followed the instructions of copying it manually with `cp` from the read-me file.

I did use `qmake` to generate the make file and the build was fine; it's running just now.

Had to install both `qt` (I pressume that is 4.x) and `qt5` from homebrew, because it seems only `qt` contains `qmake.

Rather than going through another build, I looked at GhostWriter's Makefile install target and it's empty. FocusWriter's `.pro` file doesn't contain any instructions for where to install on a Mac.

I did it again with `sudo cp -R ./build/release/ghostwriter.app /Applications` and it worked. So I'd update the instructions and see how that goes in the `.pro` file.
